packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -49,56 +54,122 @@ export function extractLogSafeErrorProperties(error: any) {
     return safeProps;
 }
 
-/** Type guard for ILoggingError */
-export const isILoggingError = (x: any): x is ILoggingError => typeof x?.getTelemetryProperties === "function";
+/** Copy props from source onto target, overwriting any keys that are already set on target */
+function copyProps(target: unknown, source: ITelemetryProperties) {
+    Object.assign(target, source);
+}
 
-/**
- * Read-Write Logging Error.  Not exported.
- * This type alias includes addTelemetryProperties, and applies to objects even if not instanceof LoggingError\
- */
-type RwLoggingError = LoggingError;
+/** Metadata to annotate an error object when annotating or normalizing it */
+export interface FluidErrorAnnotations {
+    /** Telemetry props to log with the error */
+    props?: ITelemetryProperties;
+    /** fluidErrorCode to mention if error isn't already an IFluidErrorBase */
+    errorCodeIfNone?: string;
+}
+
+/** Simplest possible implementation of IFluidErrorBase */
+export class SimpleFluidError implements IFluidErrorBase {
```

Comment:
>export [](http://example.com/codeflow?start=0&length=6)

why export this?

Reply:
I went back and forth, happy to not.

Reply:
Ehhh I might need it in the tests (or I can hack something for the tests)